packages/react-native/ReactCommon/react/nativemodule/webperformance/NativePerformance.cpp

```diff
@@ -128,6 +137,18 @@ void NativePerformance::measure(
 #endif
 }
 
+void NativePerformance::logEvent(
```

Comment:
This shouldn't be exposed to JS because these are always emitted from native.

Reply:
I exposed them for unittests 😅

Reply:
Maybe make the name more explicit (e.g.: `testOnly_logEvent`).